src/modules/raku.rs

```diff
@@ -0,0 +1,138 @@
+use super::{Context, Module, ModuleConfig};
+
+use crate::configs::raku::RakuConfig;
+use crate::formatter::StringFormatter;
+use crate::formatter::VersionFormatter;
+
+/// Creates a module with the current raku version
+pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
+    let mut module = context.new_module("raku");
+    let config: RakuConfig = RakuConfig::try_load(module.config);
+    let is_raku_project = context
+        .try_begin_scan()?
+        .set_extensions(&config.detect_extensions)
+        .set_files(&config.detect_files)
+        .set_folders(&config.detect_folders)
+        .is_match();
+
+    if !is_raku_project {
+        return None;
+    }
+
+    let parsed = StringFormatter::new(config.format).and_then(|formatter| {
+        formatter
+            .map_meta(|var, _| match var {
+                "symbol" => Some(config.symbol),
+                _ => None,
+            })
+            .map_style(|variable| match variable {
+                "style" => Some(Ok(config.style)),
+                _ => None,
+            })
+            .map(|variable| match variable {
+                "version" => {
+                    let raku_version = context
+                        .exec_cmd("raku", &["-e", "printf q/%s-%s/, $*RAKU.version, $*VM;"])?
+                        .stdout;
```

Comment:
Can you separate the different outputs into different variables, like the elixir module does? I also noticed `raku` has a `--version` flag, which might be faster and exposes information about the runtime version:
```
❯ raku --version
Welcome to Rakudo™ v2021.12.
Implementing the Raku® Programming Language v6.d.
Built on MoarVM version 2021.12.
```

Reply:
Good point @davidkna, on it! :)

Reply:
Done